app/lib/actions/getPharmacy.js

```diff
@@ -0,0 +1,18 @@
+const apiRequest = require('../etl-toolkit/apiRequest');
+const fixPhoneNumbers = require('../fixPhoneNumbers');
+const config = require('../config');
+
+function toObject(pharmaString, odsCode) {
+  const pharmacy = JSON.parse(pharmaString);
+  // eslint-disable-next-line no-underscore-dangle
+  pharmacy._id = odsCode;
+  fixPhoneNumbers(pharmacy);
```

Comment:
The db-updater is for taking a JSON file and loading it into a given mongo instance. For the gp-data ETL it was suggested that it was split into 3 parts, ETLs, combiner, and db-updater.

We thought the best place to do any data manipulation in that case was the combiner, and have the ETLs just get the records, and the db-updater just do the update.

In this case we'd need a whole new component to add the ID, fix the addresses and phone numbers which seems a lot of work. If we were combining two files, that would be the place to do it, but I'm not sure it's worth the effort in the case to add a component for the minimal processing that is done, hence taking the pragmatic decision to do it in the ETL.